pkg/planner/core/casetest/physicalplantest/testdata/plan_suite_out.json

```diff
@@ -3719,5 +3719,17 @@
         "Warning": null
       }
     ]
+  },
+  {
+    "Name": "TestPointgetIndexChoosen",
+    "Cases": [
+      {
+        "SQL": "select * from t where a=1 and b=1;",
+        "Plan": [
+          "Point_Get 1.00 root table:t, index:uab(a, b) "
```

Comment:
seems currently tidb can do the right thing for this query, any other meaningful cases?
<img width="730" alt="image" src="https://github.com/pingcap/tidb/assets/15825830/531e6f41-df5d-471c-a7ab-6fd65175a2a5">

Reply:
I already change the test to more accurate test this case